app/test/integration-test/pom.xml

```diff
@@ -36,13 +36,14 @@
 
     <syndesis.integration.runtime>spring-boot</syndesis.integration.runtime>
     <syndesis.camel.k.customizers>health,logging,syndesis</syndesis.camel.k.customizers>
-    <syndesis.logging.enabled>false</syndesis.logging.enabled>
```

Comment:
This is for debugging or do we want to keep it on?

Reply:
yes, it enables some additional logs for debugging/troubleshooting